src/mountfsd/mountwork.c

```diff
@@ -298,6 +340,7 @@ static int vl_method_mount_image(
                 { "growFileSystems",             SD_JSON_VARIANT_BOOLEAN,  sd_json_dispatch_tristate,    offsetof(MountImageParameters, growfs),               0 },
                 { "password",                    SD_JSON_VARIANT_STRING,   sd_json_dispatch_string,      offsetof(MountImageParameters, password),             0 },
                 { "imagePolicy",                 SD_JSON_VARIANT_STRING,   json_dispatch_image_policy,   offsetof(MountImageParameters, image_policy),         0 },
+                { "mountOptions",                SD_JSON_VARIANT_ARRAY,    json_dispatch_image_options,  offsetof(MountImageParameters, options),              0 },
```

Comment:
should still be a map, not an array

Reply:
just put together a jsob object with the key strings being the designator, and the values being objects with a single field: the option string.